2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -194,47 +203,70 @@ $bold("Scenario: ")$
 
 
 
-== UC-3 Modifica ambiente 3d
+= Caricamento dati database
+
+#figure(image("./imgs/uc3.png", format: "png"), caption: [UML UC-3])
+
+== Caricamento dati da database
+
 $bold("Descrizione: ")$ 
-il perimetro dell'ambiente di lavoro viene modificato successivamente alla sua configurazione iniziale.
+i prodotti vengono inseriti dal database nei rispettivi bin.
 
 $bold("Attore: ")$
 utente.
 
 $bold("Precondizioni: ")$
-- almeno una configurazione dell'ambiente deve essere avvenuta con successo;
+- l’ambiente deve essere correttamente configurato;
+- il database deve contenere almeno un prodotto con le relative coordinate.
```

Comment:
In realtà questo non deve essere forzatamente vero: posso avviare la procedura anche con una database vuoto, semplicemente non ci sarà alcun elemento da inserire. Semplicemente la query ritornerà zero elementi.